src/pages/Detail/Detail.js

```diff
@@ -0,0 +1,203 @@
+import { useState, useEffect } from "react";
+import React from "react";
+import "./Detail.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faHeart as regularHeart } from "@fortawesome/free-regular-svg-icons";
+import { faHeart as solidHeart } from "@fortawesome/free-solid-svg-icons";
+import { faShareSquare as regularShare } from "@fortawesome/free-regular-svg-icons";
+import { faChevronLeft as Left } from "@fortawesome/free-solid-svg-icons";
+import { faChevronRight as Right } from "@fortawesome/free-solid-svg-icons";
+
+function Detail() {
+  const [quantity, quantityUpdate] = useState(1);
+
+  const plus = () => {
+    quantityUpdate(quantity + 1);
+  };
+  const minus = () => {
+    if (quantity > 1) quantityUpdate(quantity - 1);
+    else {
+      alert("최소 구매 수량은 1개 이상입니다.");
+    }
+  };
+
+  const [heartshape, heartshapeChange] = useState(regularHeart);
```

Comment:
```suggestion
  const [heartShape, heartShapeChange] = useState(regularHeart);
```
camelCase 적용 부탁 드립니다~!